packages/react-menu/src/components/MenuSplitGroup/useMenuSplitGroupStyles.ts

```diff
@@ -0,0 +1,37 @@
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import type { MenuSplitGroupState } from './MenuSplitGroup.types';
+
+export const menuSplitGroupClassName = 'fui-MenuSplitGroup';
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  root: theme => ({
+    display: 'flex',
+    '& > [role="menuitem"]:nth-child(1)': {
```

Comment:
Is it safer to use `menuItemClassName` instead?

Reply:
good point, I forgot about the classnames, they're quite handy for this 👍